controllers/storagecluster/external_resources.go

```diff
@@ -326,6 +338,104 @@ func (obj *ocsExternalResources) ensureDeleted(r *StorageClusterReconciler, inst
 	return reconcile.Result{}, nil
 }
 
+// claims should be created only once and should not be created/updated again if user deletes/update it.
+func (r *StorageClusterReconciler) createDefaultStorageClassClaims(instance *ocsv1.StorageCluster) error {
+
+	storageClassClaimFile := &ocsv1alpha1.StorageClassClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      generateNameForCephFilesystemSC(instance),
+			Namespace: instance.Namespace,
+			Labels: map[string]string{
+				defaultStorageClassClaimLabel: "true",
+			},
+		},
+		Spec: ocsv1alpha1.StorageClassClaimSpec{
+			Type: "sharedfilesystem",
+		},
+	}
+
+	storageClassClaimBlock := &ocsv1alpha1.StorageClassClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      generateNameForCephBlockPoolSC(instance),
+			Namespace: instance.Namespace,
+			Labels: map[string]string{
+				defaultStorageClassClaimLabel: "true",
+			},
+		},
+		Spec: ocsv1alpha1.StorageClassClaimSpec{
+			Type: "blockpool",
+		},
+	}
+
+	err := r.createAndOwnStorageClassClaim(instance, storageClassClaimFile, storageClassClaimBlock)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (r *StorageClusterReconciler) createAndOwnStorageClassClaim(
+	instance *ocsv1.StorageCluster, claims ...*ocsv1alpha1.StorageClassClaim) error {
+
+	for _, claim := range claims {
```

Comment:
@iamniting Why do we want back to the for loop? This basically undo the request I had on my last review

Reply:
I removed it and made calls twice